stdlib/stdlib.fc

```diff
@@ -441,15 +440,15 @@ builder store_ref(builder b, cell c) asm(c b) "STREF";
 ;;; Stores `slice` [s] into `builder` [b]
 builder store_slice(builder b, slice s) asm "STSLICER";
 
-;;; Stores (serializes) an integer [x] in the range `0..2^120 − 1` into `builder` [b].
+;;; Stores (serializes) an integer [x] in the range `0..2^128 − 1` into `builder` [b].
```

Comment:
wait, `VarUInt16`'s max value is represented as `15` for the bits part and `2^(8*15)`-bit unsigned int

Reply:
Yes, you're right. I took the stdlib from [here](https://github.com/ton-community/mintless-jetton/blob/main/contracts/stdlib.fc). Seems like a mistake from their side.